misc/python/materialize/mzbuild.py

```diff
@@ -910,6 +964,12 @@ def install_arguments(parser: argparse.ArgumentParser) -> None:
             default=ui.env_is_truthy("CI_COVERAGE_ENABLED"),
             action="store_true",
         )
+        parser.add_argument(
+            "--sanitizer",
+            help="whether to enable a sanitizer (address, thread, leak, memory)",
```

Comment:
Recommend using `choices` here to enforce valid options (and then you can avoid enumerating them by hand in the help string too).